jina/executors/indexers/keyvalue.py

```diff
@@ -62,7 +62,7 @@ def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
             self.write_handler.header.write(
                 np.array(
                     (key, p, r, r + l),
-                    dtype=np.int64
+                    dtype=np.object
```

Comment:
Sorry for not being completely concrete in the first place. I meant:
```suggestion
                    dtype=[np.object, np.int64, np.int64, np.int64]
```